vnext/src/Libraries/Pressability/Pressability.windows.js

```diff
@@ -466,6 +467,7 @@ export default class Pressability {
         const {onBlur} = this._config;
         if (onBlur != null) {
           onBlur(event);
+          this._isKeyDown = false;
```

Comment:
We'll need to move this up out of the if block. We want the `onBlur` event to reset the state regardless of whether the Pressability caller has an onBlur handler.